### ISSUE TITLE
Truncated git history

### ISSUE THREAD

**Issue by philn** [Igalia/igalia-git-stats#3]
This repo seems to be a fork of https://github.com/tomgi/git_stats but the upstream git log was lost. This is unfortunate. Usually forks preserve the git history.

**Comment by Gyuyoung** [comment]
yes, it would be good to keep original git history. IIRC, the git history had been kept until I transferred this repository from my account to Igalia github. Does anyone know how to restore the git history? If not, it seems that we fork git_stats again, then cherry pick our commits on top of the history.

**Comment by philn** [comment]
Perhaps you can find it in your checkout's `reflog`... No idea. Otherwise a new fork is the way to go.
